the-devastating-chickens/BangazonSite
# WEM Product Type Display Issue #1

# Description

This pull request is for the product types list feature.

Fixes Issue # [(1)](https://github.com/the-devastating-chickens/BangazonSite/issues/1)

## Type of change


# Testing Instructions

1.`git checkout wem-productTypeDisplay-1`
1. review code
1. run app and check for build errors
1. click on the ProductTypes link on the navigation bar
1. make sure each category is listed with edit, details, and delete is showing(These features may or may not work but are not a part of this pull request)
1. make sure three products are listed below each category

# Checklist:
